encode_test.go

```diff
@@ -95,17 +115,17 @@ func toFromJSON(t *testing.T, in interface{}, recvr interface{}) {
 
 func TestNilEncodings(t *testing.T) {
 	// make sure sigs are okay with nil
-	a, b := SignatureS{}, SignatureS{}
+	a, b := Signature{}, Signature{}
```

Comment:
Yes, that is more idiomatic. Thanks

How to stress do not use except by changing the names? I'm open to suggestions